content/post/085-announcing-rust-ipfs.md

```diff
@@ -0,0 +1,61 @@
+---
+date: 2020-03-16
+url: 2020-03-16-announcing-rust-ipfs
+title: Announcing Rust IPFS
+author: Mark Robert Henderson and Volker Mische
+---
+
+![Rust IPFS Logo](https://github.com/ipfs-rust/logo/raw/master/rust-ipfs-logo-256w.png)
+
```

Comment:
Calling all rustaceans, rustafarians, ferrosities, and rustlers - we’ve got an exciting update! Active full time work on a rust-IPFS implementation has commenced, building on the great work by Parity in rust-libp2p! Equilibrium Labs is spearheading the new community and implementation with support from Protocol Labs, and is looking for additional rust devs itching to help build a new language implementation of the InterPlanetary File System combining the performance and resource utilization benefits of Rust with a keen eye on conformance you the IPFS spec.

Reply:
Added that in. Great word play :)